src/components/PersonalSettings.vue

```diff
@@ -442,7 +442,7 @@ export default {
 					showSuccess(t('integration_google', 'Google options saved'))
 					// callback
 					if (callback) {
-						callback()
```

Comment:
This seems like it would break things. Is this intentional?

Reply:
The `callback()` function is replaced by `callback(response)`. The `response` var is used on line 836 below and it updates `state.free_space` ( and UI ) to handle the case when a user switches between folders located on different storage back-ends and thus might have different amount of free space.

Reply:
ooh, right. I was confused :see_no_evil: I thought you'd removed the argument :D